docs-client/src/containers/MethodPage/RequestBody.tsx

```diff
@@ -38,6 +43,24 @@ const RequestBody: React.FunctionComponent<Props> = (props) => (
     </Button>
     {props.requestBodyOpen && (
       <>
+        {props.exampleRequests.length > 0 && (
+          <>
+            <Typography variant="body2" paragraph />
+            <Select
+              fullWidth
+              displayEmpty
+              value=""
+              renderValue={() => 'Select example requests...'}
+              onChange={props.onSelectedRequestBodyChange}
+            >
+              {props.exampleRequests.map((body) => (
+                <MenuItem key={body} value={body}>
+                  {truncate(body, 30)}
```

Comment:
I believe some users still want to see the whole body before selecting an item.
How about wrapping the item with a `Tooltip`?
```java
<Tooltip title={body} placement="right">
  <MenuItem key={body} value={body}>
    {truncate(body, 30)}
  </MenuItem>
</Tooltip>
```

Reply:
It looks like a long text renders well. https://mui.com/material-ui/react-tooltip/#variable-width